module/behaviour/strategy/SoccerStrategy/src/SoccerStrategy.cpp

```diff
@@ -139,6 +140,7 @@ namespace module::behaviour::strategy {
         on<Trigger<KillGetup>>().then([this] { isGettingUp = false; });
 
         on<Trigger<Penalisation>>().then([this](const Penalisation& selfPenalisation) {
+            emit(std::make_unique<ResetRawSensors>());
```

Comment:
Is this needed for moving arms out of zombie?

Reply:
Yes, this is needed when the robot gets penalised and reset, it gets spawned in zombie so we need to reset the sensor states so we can know to move the arms out of zombie.